src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5098,7 +5098,7 @@ Unify Enum declarations with other block type declarations.
         <description>The "As" keyword.</description>
       </child>
             
-      <child name="Type" kind="@TypeSyntax">
+      <child name="Type" optional="true" kind="@TypeSyntax">
         <description>The type-name part of the tuple element syntax.</description>
       </child>
```

Comment:
Type is actually allowed without "As". 

Dim x as (Integer(), List(of Byte))

---
In reply to: [82035049](https://github.com/dotnet/roslyn/pull/14284#discussion_r82035049) [](ancestors = 82035049)